README.md

```diff
@@ -1,3 +1,8 @@
+> [!CAUTION]
+> This codelab is deprecated and will be removed soon.
+>
+> Please see https://github.com/google-developer-training/basic-android-kotlin-compose-training-workmanager
```

Comment:
How about pointing to the codelab's content instead? https://developer.android.com/codelabs/basic-android-kotlin-compose-workmanager

Reply:
Done!!